pkg/ccl/backupccl/testdata/backup-restore/in-progress-imports

```diff
@@ -0,0 +1,469 @@
+# This test ensures that database and cluster backups properly
```

Comment:
meta note: we're starting 6 servers in this test which could make it quite a slow test, I don't know if theres anything we can do about it given the lack of a "wipe everything" tool for cluster restore but we should run this under stress a few times.